Projects/hCoreUI/Sources/UIView+TitleWordmarkView.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import UIKit
+
+extension UIView {
+    public static var titleWordmarkView: UIImageView {
```

Comment:
is this the hedvig logo? can titleWordmark might be a bit ambiguous for future reference